### PR TITLE
Styling of editor

### PR DESCRIPTION
**Pull request by bramkragten**
image

### CODE REVIEW

File: compact-custom-header-editor.js [hunk 1]
```diff
@@ -127,8 +127,9 @@ ${navigator.userAgent}
           margin-bottom: 0;
         }
         paper-button {
+          margin-top: 5px;
           background-color: var(--primary-color);
-          margin-left: 0;
```
Comment (maykar) [inline compact-custom-header-editor.js:131]: Without this the button is indented by 5px and the left hand side does not line up with any other element.

File: compact-custom-header-editor.js [hunk 2]
```diff
@@ -446,10 +480,6 @@ export class CchConfigEditor extends LitElement {
         paper-toggle-button {
           padding-top: 16px;
         }
-        paper-button {
-          background-color: var(--primary-color);
```
Comment (maykar) [inline compact-custom-header-editor.js:450]: Without this, the button is transparent and indented 5px like above.

**Comment by iantrich** [comment]
not a fan of the side-by-side on mobile, personally. was planning on stripping it from core editors

**Comment by maykar** [comment]
I think it looks quite nice on mobile.

Screenshot 1

**Comment by iantrich** [comment]
you must have a larger phone than me ;)

**Comment by maykar** [comment]
Probably :P. In testing though, it still looks good till about 350px wide then text wraps start looking a bit strange, but still usable.

File: compact-custom-header-editor.js (hunk 2 above)
Comment (bramkragten) [reply compact-custom-header-editor.js:450]: Yeah, I figured that button shouldn’t get that much attention...

**Pull request merged**